src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Each side of a triangle should not be less than 0');
+    }
 
-export class Circle {
+    if (a >= (b + c) || b >= (a + c) || c >= (b + a)) {
+      throw new Error('These sides cannot form a triangle');
+    }
+  }
 
+  getArea(): number {
+    const semiPer = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(
+      semiPer * (semiPer - this.a) * (semiPer - this.b) * (semiPer - this.c),
+    );
+
+    return Math.floor(area * 100) / 100;
```

Comment:
DRY
You make it 3 times, so, move to a separate function & reuse it
```suggestion
    return getRounding(area);
```